tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -195,6 +196,7 @@ _gen_kernel_bin_rule = rule(
         "gpu_archs": attr.string_list(),
         "jit": attr.bool(mandatory = False),
         "cpu_codegen": attr.bool(mandatory = False),
+        "index_64bit": attr.bool(mandatory = False),
```

Comment:
This should also be something like `jit_i64_indexed_for_large_tensors`.

Reply:
done!